### PR TITLE
Fix missing method bug in cohort_csv, and add tests 

### PR DESCRIPTION
**Pull request by aoby**
…to ensure all _csv methods are called and don't raise errors

### CODE REVIEW

**Comment by aoby** [comment]
[Fix download cohort view as CSV](https://trello.com/c/UYHk2qsm/516-fix-download-cohort-view-as-csv)

File: dashboard/test/models/pd/application/teacher1819_application_test.rb [hunk 1]
```diff
@@ -756,13 +756,31 @@ class Teacher1819ApplicationTest < ActiveSupport::TestCase
     end
 
     test 'locked status appears in csv only when the supplied user can_see_locked_status' do
+      application = build :pd_teacher1819_application
       mock_user = mock
 
       Teacher1819Application.stubs(:can_see_locked_status?).returns(false)
```
Comment (clareconstantine) [inline dashboard/test/models/pd/application/teacher1819_application_test.rb:762]: based on the variables below (`header_with_locked`, `row_with_locked`), this stub should return `true`

File: dashboard/test/models/pd/application/teacher1819_application_test.rb [hunk 2]
```diff
@@ -756,13 +756,31 @@ class Teacher1819ApplicationTest < ActiveSupport::TestCase
...
-      refute Teacher1819Application.csv_header('csf', mock_user).include? 'Locked'
+      header_with_locked = Teacher1819Application.csv_header('csf', mock_user)
+      refute header_with_locked.include? 'Locked'
+      row_with_locked = application.to_csv_row(mock_user)
+      assert_equal CSV.parse(header_with_locked).length, CSV.parse(row_with_locked).length,
+        "Expected header and row to have the same number of columns, including Locked"
 
       Teacher1819Application.stubs(:can_see_locked_status?).returns(true)
```
Comment (clareconstantine) [inline dashboard/test/models/pd/application/teacher1819_application_test.rb]: And this stub should return `false`

File: dashboard/test/models/pd/application/teacher1819_application_test.rb (hunk 1 above)
Comment (aoby) [reply dashboard/test/models/pd/application/teacher1819_application_test.rb:762]: D'oh! Actually the variables are named wrong. The assertions match the stubs, but the variable names don't. Will fix.

**Pull request merged**